docs/getting-started.md

```diff
@@ -216,16 +233,17 @@ Troubleshooting tips if your EV3 won't boot
 
 ## Step 5: Set up a network connection
 
-There are lots of choices here. Choose the connection option that best suits you
-and your available devices.
+Click on the button below to visit our networking page. There are lots of
+choices here; choose the connection option that best suits you and your
```

Comment:
For me - yes, only because we don't use such contructions anywhere else (not because it is incorrect). But I had a feeling "something is not right here?"

If I had to I would rather use hyphen - instead of ; but it is totally subjective and I am non native in English so don't listen ;-)